test/test_torch.py

```diff
@@ -18859,6 +18859,10 @@ def test_dstack(self, device, dtype):
             expected = np.dstack(np_input)
             self.assertEqual(actual, expected)
 
+    def test_mean_repeated_dim(self, device):
+        x = torch.randn(3, 3, 3, 3, device=device)
+        with self.assertRaisesRegex(RuntimeError, r'mean: repeated dimension in `dim` \(\[0, 0\]\)'):
+            torch.mean(x, dim=(0, 0))
```

Comment:
@mruberry So, as was mentioned in the original issue, there are a bunch of operators which take in a list of dimensions. It seems like it would be useful to easily run a version of this test for all of the operators that do this :>

Reply:
Good idea. I've made a note.

Reply:
I had quickly searched the file below
https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml
for `int[1]` based on the signature of https://github.com/pytorch/pytorch/blob/cce5982c4ceb77a0797e8bd4c717ebfec0681eab/aten/src/ATen/native/native_functions.yaml#L2093